guides/doc-Release_Notes/topics/katello-4.6.0.adoc

```diff
@@ -0,0 +1,100 @@
+* Add Deb Repositories and Packages Tab in Lifecycle Environment (https://projects.theforeman.org/issues/34793[#34793], https://github.com/Katello/katello/commit/edef00619ae6a6f393d99ef2012353e684681022[edef0061])
+* As a user, I can CRUD Simplified ACSs via the API (https://projects.theforeman.org/issues/34336[#34336], https://github.com/Katello/katello/commit/95f66df3a7382f65b151617edb1aa943dd41a195[95f66df3])
+
+### Other
+* CVV Compare - Add the "View By - Different/Same" dropdown (https://projects.theforeman.org/issues/35323[#35323], https://github.com/Katello/katello/commit/a5759f7e960ab76f179d30d7d7af7d808a2bf9c7[a5759f7e])
+* Empty Searches - adding link to clear search (https://projects.theforeman.org/issues/35027[#35027], https://github.com/Katello/katello/commit/37899ec73edf2bdd4f9e333075c9c240e506875d[37899ec7])
+* Drop use of pulp_client certificates (https://projects.theforeman.org/issues/35004[#35004], https://github.com/Katello/katello/commit/63a1499fe319588bed8006cffada4faccd13a532[63a1499f])
+* Update pulp_deb to 2.18.0 (https://projects.theforeman.org/issues/34829[#34829], https://github.com/Katello/katello/commit/65c23872583555d41a6d846e39e8e8d287fe566a[65c23872])
+
+## Bug Fixes
+
+### Hosts
+* useTableSort should translate initialSortColumnName (https://projects.theforeman.org/issues/35329[#35329], https://github.com/Katello/katello/commit/dcdd1e061bc70d31083b44116f4cedd00d96796e[dcdd1e06])
+* new host ui, content, errata subtab,  when N/A is chosen as severity filter erratas results are empty (https://projects.theforeman.org/issues/35277[#35277], https://github.com/Katello/katello/commit/3da0b394c9bb7d76e20547d179884bc7826a68e7[3da0b394])
+* "Registered Content Hosts" Report is Showing the Wrong Available Kernel Version for RHEL 7.7 Client (https://projects.theforeman.org/issues/35040[#35040], https://github.com/Katello/katello/commit/f98327e10b5cbadfd0e060ba6b8fe468bbcc706c[f98327e1])
+* Overview tab (new host details) - Correct card order (https://projects.theforeman.org/issues/35002[#35002], https://github.com/Katello/katello/commit/213530172e0ca8ac0df7b36b221a804e980bf78d[21353017])
+* Errata overview card - Add tooltips for errata type icons (https://projects.theforeman.org/issues/34769[#34769], https://github.com/Katello/katello/commit/064a710f48d7118f4db6a039be73f2d655d33ee1[064a710f])
+* host registration form should include activation key selection on the first page (https://projects.theforeman.org/issues/33902[#33902], https://github.com/Katello/katello/commit/8ba12c34f521651a243be80ac77675bd070e6eed[8ba12c34])
+
+### Repositories
+* Katello rpm search via nvra also (https://projects.theforeman.org/issues/35290[#35290])
+* b'Not able to enable repositories when FIPS is enabled.' (https://projects.theforeman.org/issues/35262[#35262], https://github.com/Katello/katello/commit/66d9e4e7291d1041b5c2da3d22f10de773e77f18[66d9e4e7])
```

Comment:
I updated this issue to drop the `b''` which comes from using Python 3 with old code.